packages/admin/src/table/filterbar/FilterBarPopOverFormField.tsx

```diff
@@ -0,0 +1,187 @@
+import { Box, Button, ButtonProps, Popover, Typography } from "@material-ui/core";
+import { makeStyles, Theme, ThemeOptions, useTheme } from "@material-ui/core/styles";
+
+import get = require("lodash.get");
```

Comment:
please convert to import

Reply:
i did that because of we already used that syntax in the useTbaleQueryFilter.tsx File for the isEqual lodash function :)